src/components/Tool/MergeContacts/MergeContacts.test.tsx

```diff
@@ -0,0 +1,208 @@
+              state: 'FL',
+              postalCode: '32832',
+              source: 'MPDX',
+            },
+          },
+          recordTwo: {
+            id: 'contact-4',
+            avatar: 'https://mpdx.org/images/avatar.png',
+            name: 'Doe, Jane and Paul',
+            createdAt: '1999-04-02T00:00:00-05:00',
+            status: StatusEnum.NeverContacted,
+            primaryAddress: {
+              id: 'address-2',
+              street: '123 Leonard Ave',
+              city: 'Orlando',
+              state: 'FL',
+              postalCode: '32832',
+              source: 'MPDX',
+            },
+          },
+        },
+      ],
+    },
+  },
+};
+
+const mockEnqueue = jest.fn();
+
+jest.mock('notistack', () => ({
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  ...jest.requireActual('notistack'),
+  useSnackbar: () => {
+    return {
+      enqueueSnackbar: mockEnqueue,
+    };
+  },
+}));
+
+interface MergeContactsWrapperProps {
+  mutationSpy?: () => void;
+}
+
+const MergeContactsWrapper: React.FC<MergeContactsWrapperProps> = ({
+  mutationSpy,
+}) => {
+  return (
+    <ThemeProvider theme={theme}>
+      <TestRouter>
+        <GqlMockedProvider<{
+          GetContactDuplicates: GetContactDuplicatesQuery;
+        }>
+          mocks={mocks}
+          onCall={mutationSpy}
+        >
+          <ContactsProvider
+            activeFilters={{}}
+            setActiveFilters={() => {}}
+            starredFilter={{}}
+            setStarredFilter={() => {}}
+            filterPanelOpen={false}
+            setFilterPanelOpen={() => {}}
+            contactId={[]}
+            searchTerm={''}
+          >
+            <MergeContacts accountListId={accountListId} />
+          </ContactsProvider>
+        </GqlMockedProvider>
+      </TestRouter>
+    </ThemeProvider>
+  );
+};
+
+describe('Tools - MergeContacts', () => {
+  it('should render', async () => {
+    const { findByText, getByTestId } = render(<MergeContactsWrapper />);
+
+    expect(await findByText('Merge Contacts')).toBeInTheDocument();
+    expect(getByTestId('ContactMergeDescription').textContent).toMatch(
+      'You have 55 possible duplicate contacts',
+    );
+  });
+
+  // TODO
+  // it('should select clicked contact', async () => {
+  //   const { queryAllByTestId } = render(<MergeContactsWrapper />);
+
+  //   await waitFor(() =>
+  //     expect(queryAllByTestId('MassActionsMergeModalContact')).toHaveLength(2),
+  //   );
+
+  //   const contacts = queryAllByTestId('MassActionsMergeModalContact');
+  //   expect(queryByText(contacts[0], 'Use This One')).toBeInTheDocument();
+  //   expect(queryByText(contacts[1], 'Use This One')).not.toBeInTheDocument();
+
+  //   userEvent.click(contacts[1]);
+
+  //   expect(queryByText(contacts[0], 'Use This One')).not.toBeInTheDocument();
+  //   expect(queryByText(contacts[1], 'Use This One')).toBeInTheDocument();
+  // });
+
+  it('should merge contacts', async () => {
```

Comment:
On this test you should check that the contacts have been removed from this page post merge.

Reply:
The page automatically queries the list of duplicate contacts after merging. So the mocked data is brought back in and shows the same contacts. Do you have any ideas how I could test this?

Reply:
I looked into it; I believe there will be a way somehow, but don't get stuck on it. I'm good with you moving on without it, as it's turning into a blocker. We can come back and fix it at a later date.